exonum-java-binding/packaging/pom.xml

```diff
@@ -16,11 +16,12 @@
   <description>A fake module used for packaging EJB App</description>
 
   <properties>
-    <!--A directory used for storing immediate files while packaging the App-->
+    <!--A directory used for storing temporary files while packaging the App-->
```

Comment:
Is there a documentation anywhere on what we place in `target/prepackage` and what in `target/debug`? Currently it's not clear why lib/java ends up in the latter; and (something) in the former. Also, why other things (like lib/java) are placed _outside_ of a *base*dir?

Reply:
`target/debug` is filled by Cargo and therefore contains only libjava_bindings.so. In `target/prepackage` we temporarily store licenses and libstd.so. `target/debug/lib/native` and `target/debug/lib/java` are both required for running EJB App from `target` in our scripts. Their content will be copied to `lib/java` and `lib/native` inside App archive. 

As some kind of improvement we can store `libstd` inside `lib/native` as well, though it's not necessary. Also we can move `prepackage` to `target/debug` - it would be more consistent.

Reply:
Made some improvements - now prepackage directory is replaced by the new one `target/debug/etc`. The whole directory structure is explained in package_app.sh script

Reply:
Are you saying

```
target/debug/prepackage/
                       - lib
                            + java
                            + rust
```
?

Reply:
No, without prepackage.